tests/hello/assembly/main.ts

```diff
@@ -8,6 +8,15 @@ export function hello(name: string): string {
   return "hello " + name;
 }
 
+export function setKeyValue(key: string, value: string): string {
```

Comment:
You should switch it to ": void {". Don't need to return value here

Reply:
Done